theseus/embodied/collision/collision.py

```diff
@@ -23,11 +23,12 @@ def __init__(
         sdf_data: Variable,
         sdf_cell_size: Variable,
         cost_eps: Variable,
+        loss_function: Optional[LossFunction] = None,
```

Comment:
What's the behavior if we don't add this here. The super call would not have anything so the base class would assign trivial loss?

Reply:
That's correct. We can skip `loss_function`, and then `collision` uses `TrivialLoss`.